pkg/controller/migmigration/pod.go

```diff
@@ -377,9 +224,44 @@ func (t *Task) veleroPodCredSecretPropagated(cluster *migapi.MigCluster) (bool,
 				return false, nil
 			}
 		} else {
+
+			return false, nil
+		}
+	}
+
+	return true, nil
+}
+
+// Ensure the stage pods have been deleted.
+func (t *Task) ensureStagePodsTerminated() (bool, error) {
+	client, err := t.getDestinationClient()
+	namespaceList := t.destinationNamespaces()
+	if err != nil {
+		log.Trace(err)
+		return false, err
+	}
+	cLabel, _ := t.Owner.GetCorrelationLabel()
+	podList := corev1.PodList{}
+	for _, ns := range namespaceList {
+		options := k8sclient.InNamespace(ns)
+		err := client.List(context.TODO(), options, &podList)
+		if err != nil {
+			return false, err
+		}
+		for _, pod := range podList.Items {
+			// Owned by ANY migration.
```

Comment:
Same question as above

Reply:
Ah, missed this before, Alex.
The same namespace & label selector used for _delete_ should be used here.

Reply:
Done